source/Applications/Basic/Visualization/CaptureVis3D.m

```diff
@@ -0,0 +1,38 @@
+try
+    % Adding directory that contains zividApplication and connectCamera to search path.
+    addpath(genpath([fileparts(fileparts(fileparts(pwd))),filesep,'Camera',filesep,'Basic']));
+
+    app = zividApplication;
+
+    disp('Setting up visualization');
+    visualizer = Zivid.NET.CloudVisualizer();
+    app.DefaultComputeDevice = visualizer.ComputeDevice;
+
+    disp('Connecting to camera')
+    camera = connectCamera(app);
+
+    disp('Adjusting the iris');
+    settings = camera.Settings;
+    settings.Iris = 20;
+    camera.SetSettings(settings);
+
+    disp('Capture a frame');
+    frame = camera.Capture;
+
+    resultFile = 'result.zdf';
+    disp(['Saving frame to file: ' resultFile]);
+    frame.Save(resultFile);
+
+    disp('Display the frame');
+    visualizer.Show(frame);
```

Comment:
Be aware that the visualizer can only be started once in Matlab, so when you close it you have to shut down matlab to run this script again. So might be a though not to samples for visualizer?

Reply:
Is that a limitation of Matlab handling .NET or lmitation of our SDK? If it is not our limitation, I am thinking we can comment on it in the sample?

Reply:
I think it is in our SDK as well. But normally in NET you would in theory exit when finished with the scripts. So its not seen in the sample, but its there and Stian wants to fix it.

The issue is that you can't run CaptureVis3D twice in same instance. So visualizer is basically useless in matlab. So might be worth not having thode samples.

Reply:
Decided to keep it, in addition to this printed out:
Note: It is not possible to run the Zivid visualizer repeatedly in Matlab. You have to restart Matlab to call the visualizer again.